### PR TITLE
Add Gurusewak's newyork file to repo

### LINKED ISSUE #11: Gurusewak's workflow for adding New York to the repo
## New York
- [x] Create an issue
- [x] Create a branch
- [x] Make changes 
- [x] Open a pr
- [x] Collaborate
- [ ] Get approval 
- [ ] Merge

### PR DESCRIPTION
**Pull request by gurusewak**
closes #11

### CODE REVIEW

File: gurusewak-newyork.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## New York 
+
+### Restaurants
+Halal Guys
+McSorleys
```
Comment (alanrhannah) [inline gurusewak-newyork.md:5]: There is a McSorleys in Glasgow, UK!

File: gurusewak-newyork.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Kathi Roll Company
+
+### Places to go 
+Times Sq
+Empire State
+Freedom Tower
+Brooklyn Bridge
+SOL
+
```
Comment (alanrhannah) [inline gurusewak-newyork.md:15]: This would cool as a list...

File: gurusewak-newyork.md (hunk 1 above)
Comment (githubteacher) [reply gurusewak-newyork.md:5]: I didn't know that! That's awesome!

**Comment by gurusewak** [comment]
@githubteacher good to 🚢 ?

**Review by BugmanMaster — APPROVED** [review]
HArd to find this...

**Pull request merged**